README.md

```diff
@@ -69,6 +72,10 @@ Other helpful methods:
 * `Beaker::TestmodeSwitcher.mode`: Returns the currently configured test mode.
 * `Beaker::TestmodeSwitcher.runner`: Returns the currently configured runner.
 
+## How to use on a multi node set up
+Ensure that you document all of the nodes you would like TestmodeSwitcher to run on in the hosts file `/spec/acceptance/nodesets`.
+When you then use `execute_manifest_on(hosts, manifest, opts={})` puppet will do an agent run on all hosts available within the hosts file.
```

Comment:
`execute_manifest_on` will execute the manifest on all `hosts`, not on all hosts in the nodeset. You can also assume here that people know how to setup their nodeset.

Since there is not yet any way to trigger multinode tests, there is not much you can say here.